apps/docs/docs/reference/ui/molecules/RadioField.mdx

```diff
@@ -28,10 +28,6 @@ RadioField wraps a [Radio](/reference/ui/atoms/Radio) input and its correspondin
 </div>
 ```
 
-<small>
-  You can remove <code>readOnly</code> prop when using.
-</small>
-
 ---
```

Comment:
Noticed that the Design Token is wrong. The default value should be `var(--fs-spacing-1)`. The same fix should be done on `CheckboxField `